src/scapy/layers/ssl_tls_crypto.py

```diff
@@ -107,34 +110,37 @@ def __init__(self):
         self.crypto.server.rsa = namedtuple('rsa', ['pubkey','privkey'])
         self.crypto.server.rsa.pubkey=None
         self.crypto.server.rsa.privkey=None
-        self.crypto.session = namedtuple('session', ['premaster_secret',
-                                                     'master_secret'])
+        self.crypto.session = namedtuple('session', ["encrypted_premaster_secret",
+                                                     'premaster_secret',
+                                                     'master_secret',
+                                                     "prf"])
         
         self.crypto.session.encrypted_premaster_secret=None
         self.crypto.session.premaster_secret=None
         self.crypto.session.master_secret=None
+        self.crypto.session.prf = TLSPRF(SHA256)
         self.crypto.session.randombytes = namedtuple('randombytes',['client','server'])
         self.crypto.session.randombytes.client=None
         self.crypto.session.randombytes.server=None
         
         self.crypto.session.key = namedtuple('key',['client','server'])
-        self.crypto.session.key.server = namedtuple('server',['mac','encryption','iv'])
+        self.crypto.session.key.server = namedtuple('server',['mac','encryption','iv', "seq"])
         self.crypto.session.key.server.mac = None
         self.crypto.session.key.server.encryption = None
         self.crypto.session.key.server.iv = None
+        self.crypto.session.key.server.seq_num = 0

```

Comment:
the property was registered as "seq" 4 lines above. "seq" => "seq_num". also see client